src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend_utils.py

```diff
@@ -50,11 +51,16 @@ def _get_cache_dir(options) -> Optional[Any]:
 
 def _get_model_caching(options) -> Optional[Any]:
     if options is not None and "model_caching" in options:
-        return options["model_caching"]
+        caching = options["model_caching"]
+        if bool(caching) and str(caching).lower() not in ["false", "0"]:
+            return True
+        else:
+            return False
     else:
         return os.getenv("OPENVINO_TORCH_MODEL_CACHING")
```

Comment:
I think this branch also should be processed to return bool value. os.getenv returns string in case if variable provided and None if not

Reply:
Fixed this